esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -788,43 +789,87 @@ def getSchedule(program, user, schedule_type=None, room_numbers=True):
 
         schedule = u''
         if schedule_type in [u'Student', u'Teacher']:
-            if room_numbers:
-                schedule = u"""
-    %s schedule for %s:
-
-     Time                   | Class                                  | Room\n""" % (schedule_type, user.name())
-            else:
-                schedule = u"""
-    %s schedule for %s:
 
-     Time                   | Class                                  \n""" % (schedule_type, user.name())
-            schedule += u'------------------------+---------------------------------------------------\n'
             if schedule_type == u'Student':
                 classes = ProgramPrintables.get_student_classlist(program, user)
             elif schedule_type == u'Teacher':
                 classes = ProgramPrintables.get_teacher_classlist(program, user)
+
+            schedule = """
+                        <p> %s schedule for %s: </p>
```

Comment:
I know it's not in the original version, but would it be valuable to specify the program the schedule is for as well?